lxd/operations/operations.go

```diff
@@ -128,6 +134,9 @@ type Operation struct {
 
 // OperationCreate creates a new operation and returns it. If it cannot be
 // created, it returns an error.
+//
+// Operations created inside handlers should be returned as an operationResponse
+// and should have r set to the request that is being handled.
```

Comment:
How about this:

> Operations which are returned as an operationResponse to the API handler should have `r` set to the request that is being handled so that its result can be accounted for in the metrics.